src/Settings.cpp

```diff
@@ -46,7 +47,7 @@ void OsqpEigen::Settings::setAdaptiveRho(const bool isRhoStepSizeAdactive)
 # if EMBEDDED != 1
     m_settings->adaptive_rho = (c_int)isRhoStepSizeAdactive;
 # else
-    std::cerr<< "[OsqpEigen::Settings::setAdaptiveRho] OSPQ has been set to EMBEDDED, hence this setting is disabled." << std::endl;
+    debugStream<< "[OsqpEigen::Settings::setAdaptiveRho] OSPQ has been set to EMBEDDED, hence this setting is disabled." << std::endl;
```

Comment:
I guess that some ``()`` are missing on this file.

Reply:
They are! Fixed now.